### ISSUE TITLE
Windows 11 clipboard history: NVDA announces live regions when closing clipboard history interface with items present

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#17308]
Note: the issue described was mitigated via an add-on (explained below).

### Steps to reproduce:
Prerequisite: Windows 11

1. Copy items to the clipboard, preferably at least two items.
2. Press Windows+V to open clipboard history. Make sure at least one item is present in the history list.
3. Close clipboard history (Escape).

### Actual behavior:
NVDA announces "pinned item" after closing clipboard history.

### Expected behavior:
NVDA does not announce clipboard history interface after closing it.

### NVDA logs, crash dumps and other attachments:
N/A

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-34344,64ba6c69

#### Windows version:
Windows 11 24H2 dev (build 26120.2130)

#### Name and version of other software in use when reproducing the issue:
None

#### Other information about your system:
Used as a development workstation

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Reproducible with 2024.3.x and 2024.4 beta/RC

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not applicable

### Technical information
If there are items in clipboard history, it raises live region change event (coming from data item control) which is caught and announced by NVDA. This issue was reported to Microsoft. In the meantime, Windows App Essentials add-on provides a fix by telling NVDA to ignore live region change event upon object construction.

Thanks.
